.cicd/test_manager.py

```diff
@@ -0,0 +1,68 @@
+# #######
+# Copyright (c) 2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from integration_tests.tests.test_cases import PluginsTest
+from integration_tests.tests import utils as test_utils
+
+PLUGIN_NAME = 'cloudify-openstacksdk-plugin'
+
+
+class OpenstackPluginTestCase(PluginsTest):
+
+    base_path = os.path.dirname(os.path.realpath(__file__))
+
+    @property
+    def plugin_root_directory(self):
+        return os.path.abspath(os.path.join(self.base_path, '..'))
+
+    @property
+    def client_config(self):
+        return {
+            'auth_url': os.getenv('openstack_auth_url'),
+            'username': os.getenv('openstack_username'),
+            'password': os.getenv('openstack_password'),
+            'project_name': os.getenv('openstack_project_name'),
+            'region_name': os.getenv('openstack_region_name'),
+        }
+
+    def check_main_blueprint(self):
+        blueprint_id = 'manager_blueprint'
+        self.inputs = dict(self.client_config)
+        self.inputs.update(
+            {
+                'external_network_id': 'dda079ce-12cf-4309-879a-8e67aec94de4',
```

Comment:
@EarthmanT do you think we should update `external_network_id` as environment variable